tools/clang/test/DXC/ftime-trace.hlsl

```diff
@@ -1,6 +1,5 @@
 // RUN: %dxc -E main -T vs_6_0 %s -ftime-trace | FileCheck %s
-// RUN: %dxc -E main -T vs_6_0 %s -ftime-trace=%t.json
-// RUN: cat %t.json | FileCheck %s
+// RUN: %dxc -E main -T vs_6_0 %s -ftime-trace=%t.json && FileCheck %s --input-file=%t.json
```

Comment:
This change is fine, but it also seems like you’re trying to remove a dependency on `cat`. I worry about this with upstream LLVM where tests are allowed to use `cat`

Reply:
This is trying to unblock one branch pipeline temporarily.
Once we get updated VM which can run git usr bin, we can switch it back.

Reply:
I think a comment should explain that then and these changes should be mentioned in the description if not the title since it only refers to the pdb naming changes.